### PR TITLE
Material

### LINKED ISSUE #115: Material >=10.2.0
`ng update @angular/material`
🔻
`Package "@mat-datetimepicker/core" has an incompatible peer dependency to "@angular/cdk" (requires "~10.1.1", would install "10.2.3").`
`× Migration failed: Incompatible peer dependencies found.`

### PR DESCRIPTION
**Pull request by Eugeno**
Resolves #115

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,4 +61,4 @@
     "typescript": "3.9.7",
     "webpack-bundle-analyzer": "3.8.0"
   }
-}
+}
```
Comment (bbortt) [inline package.json:64]: that's kind of bad practice

File: tsconfig.json [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
+      ],
+      "@mat-datetimepicker/moment/*": [
+        "dist/moment/*"
+      ]
     }
-  ]
-}
\ No newline at end of file
+  }
+}
```
Comment (bbortt) [inline tsconfig.json:37]: aight, I'm fine with this.

**Review by bbortt — APPROVED** [review]
thanks very much @Eugeno. could you test the new verison if I release a `beta`? I do not have any project using `@mat-datetimepicker` at the moment.

File: package.json (hunk 1 above)
Comment (Eugeno) [reply package.json:64]: hm, don't know why my IDE did it =)

**Comment by Eugeno** [comment]
> thanks very much @Eugeno. could you test the new verison if I release a `beta`? I do not have any project using `@mat-datetimepicker` at the moment.

**Comment by Eugeno** [comment]
yep, I can test with my projects

**Comment by Eugeno** [comment]
checked common use cases, everything seems to be ok

**Comment by bbortt** [comment]
thanks. I released `5.1.0` stable.

**Pull request merged**